docs/group_guidelines/git_standards.md

```diff
@@ -0,0 +1,86 @@
+
+### Dashes & Underscores
+To stay consistent with the way GitHub displays spaces in words, please use - in your branch name for spaces, and _ for
+the separator between the issue id and the feature description. 
+
+# Pull Requests
+
+## Creation:
+PR names and branch names will follow a similar syntax, as everything is based on issues.
+### Title:
+Pull request names should follow a similar syntax to branch names. They should start with the issue # which they aim to 
+close, and then continue on to provide a short informative summary of what the pull request accomplishes. Below is the
+naming scheme we should follow for all pull requests: 
+<issue-id> <short PR summary>
+### Description:
+Above all else, please ensure you use the closing keyword within your PR description, this ensures that the issue tagged 
+is successfully resolved. If your PR does not close an issue, explain why. Your description should explain what you did,
+why you did it, and if there are any known issues please mention them within the pull request.
+
+## Review:
+The goal when reviewing a pull request is to ensure that all code follows the standards we have defined within our code
+standards document - to be created when our tech stack is defined.
+
+When leaving comments, be sure you are providing constructive criticism, and if you request changes to the pull request,
+please describe precisely what requires change, and why it requires change.
+
+## Merging:
+As a rule of thumb, it is the responsibility of the author of the pull request to merge the code once it has been 
+approved.
+
+# Issues:
+
+Issue templates have been created for each issue label which was required for the capstone course. Additionally, a bug 
+report template has been created - when we detect a bug within the code, please create a new issue for it using the bug
+report!
+
+## Naming: 
+
+Each template has a default name following the syntax [label] When creating a new issue, please delete this, and enter 
+a brief summary of the issue. 
+
+## Descriptions: 
+
+Please be as descriptive as possible when creating issues. The template is there to guide you along a standard outline,
+however, if certain sections are not applicable, or a separate section is required, please add or delete as necessary.
+As long as the issue is detailed, that is all that matters.
+
+
+
+# Commits:
+
+## Commit Message:
+This might take a bit to get used to - but please write your commit messages in the imperative. This is a bit weird, but
+it ensures the commit messages line up with commit messages that are generated via commands such as git merge or git 
+revert. For example, use fix instead of fixed, or add instead of adds. 
+
+**A properly phrased commit message should be able to complete the following sentence in a grammatically correct 
+fashion:**
+
+*if applied, this commit will \<your commit message here>*
```

Comment:
can you please provide one more example? I'm rusty on my imperative

Reply:
also having a character limit is helpful in making those commit messages small

Reply:
I'm not entirely sure another example is super necessary - when wording your message just keep in mind that it needs to complete the sentence on line 81 in a grammatically correct fashion. Let me know if you'd still like another example!